src/V3Order.cpp

```diff
@@ -615,6 +615,43 @@ class MTaskVxIdLessThan final {
     }
 };
 
+//######################################################################
+// Gather variables written by a tree
+
+class OrderGatherWrittenVisitor final : public AstNVisitor {
+    std::unordered_set<AstVarScope*> m_writtenVariables;  // Variables written under tree
+
+    virtual void visit(AstVarRef* nodep) override {
+        if (nodep->access().isReadOnly()) return;
+        AstVarScope* const varScopep = nodep->varScopep();
+        if (varScopep->varp()->isFuncLocal()) return;
+        m_writtenVariables.insert(varScopep);
+    }
+    virtual void visit(AstNode* nodep) override { iterateChildrenConst(nodep); }
+
+    explicit OrderGatherWrittenVisitor(AstNode* nodep) { iterate(nodep); }
```

Comment:
```suggestion
    explicit OrderGatherWrittenVisitor(AstNode* nodep) { iterateConst(nodep); }
```

to match above?

Reply:
Actually, there is no such method (this is just the double dispatch via accept over the given node)